Project/Source/Components/UI/ComponentText.h

```diff
@@ -7,7 +7,7 @@
 #include <array>
```

Comment:
std libraries and our libraries must have a space in between them

Reply:
Changed